docker/config/config.go

```diff
@@ -43,19 +32,22 @@ func (c *Config) IsEmpty() bool {
 func (c *Config) GetCredentials(registry string) (string, string, bool) {
 	_, defined := c.usernames[registry]
```

Comment:
```
if _, defined := c.usernames[registry]; !defined {
    return "", "", false
}
```
in cases where nobody outside the if scope has to know about `defined` value

Reply:
Yes @zmalik You are right :ok_hand: